src/plugins/workspace/public/components/workspace_creator/workspace_form.tsx

```diff
@@ -402,6 +412,12 @@ export const WorkspaceForm = ({
                     features.length > 0 ? `(${selectedIds.length}/${features.length})` : ''
                   }`}
                   checked={selectedIds.length > 0}
+                  disabled={
+                    isWorkspaceFeatureGroup(featureOrGroup)
+                      ? false
+                      : // If this feature isn't checked, will not disable to allow check to compatible dirty data.
+                        isDefaultCheckedFeatureId(featureOrGroup.id) && selectedIds.length > 0
+                  }
```

Comment:
Prefer refactor to
```ts
!isWorkspaceFeatureGroup(featureOrGroup) && isDefaultCheckedFeatureId(featureOrGroup.id) && selectedIds.length > 0
```

Reply:
Do we need to add the check for `selectedIds.length > 0`?

Reply:
The reason that I added length check here before is that I want to complicate dirty data that not checked when creating. Since I have added append logic in init stage. I think we don't need it now.